Monika After Story/game/script-topics.rpy

```diff
@@ -10351,70 +10358,174 @@ label monika_beingevil:
     $ _history_list.pop()
     menu:
         m "I just hope nobody starts thinking you're evil by being here with me.{fast}"
+        "You haven't done anything wrong":
+            # variable setting at the end because they're used to determine if player has seen this path before
+            if (persistent._mas_pm_monika_innocent is None):
+                # some extra aff if this is the first choice
+                $ mas_gainAffection(amount=3, bypass=True)
+            else:
+                $ mas_gainAffection()
+
+            if (
+                (not persistent.monika_kill and mas_isMoniNormal(higher=True))
+                or (store.mas_anni.pastOneMonth() and mas_isMoniHappy(higher=True))
+            ):
+                if (renpy.random.randint(1,10) == 1) and (persistent._mas_pm_monika_innocent is not None):
```

Comment:
I'm not a fan of this one.
1. Way too rng. Not only you get the chance to see this only on the first view, but you also have to get lucky.
2. People rarely go and read pm topics again, which means most users will see only this joke, never reading the actual appreciation from Monika (that's if you remove the rng part).

If you want to keep this, maybe you could combine both paths into one?

Reply:
Hmm, you're right. Still not sure if I like this or not. I doubt many people will see this. But at least change the condition into
`persistent._mas_pm_monika_innocent is not None and renpy.random.randint(1, 10) == 1`

Reply:
Okay. Should I also raise the chance? 1 in 10 is not very high in this context, now that you mention it.

Reply:
If you want, yeah. 10% is pretty small.